README.md

```diff
@@ -24,6 +24,18 @@ optional arguments:
   -v VERSION, --version VERSION
                         Version (default=latest)
 ```
+Example:
+```bash
+# cd artifacthub
+python3 generate_config.py -r keptn-sandbox/datadog-service -v 0.2.0
+```
+Use pip to install the dependencies like this:
+```bash
+# cd artifacthub
+pip install -r requirements.txt 
+```
+The script was tested with python 3.8.10 (available by default on Ubuntu 20.04) and pip 3.8 
```

Comment:
I am not sure if this is a great idea because we might have to change the Ubuntu, python and pip version here in the future. I wanted to add it so that the user has an idea about what tooling can be used to run the script.

Reply:
I think it's okay to state what it was tested with. I wouldn't mention Ubuntu or Pip, but primarily refer to Python 3 version